<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieTab</class>
 <widget class="QWidget" name="MovieTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>740</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Movie Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleInput"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Genre:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="genreInput"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="durationInput">
        <property name="suffix">
         <string> minutes</string>
        </property>
        <property name="minimum">
         <number>60</number>
        </property>
        <property name="maximum">
         <number>240</number>
        </property>
        <property name="value">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Rating:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="ratingCombo">
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PG-13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NC-17</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="movieTable">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Genre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rating</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addMovieBtn">
       <property name="text">
        <string>Add Movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateMovieBtn">
       <property name="text">
        <string>Update Movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteMovieBtn">
       <property name="text">
        <string>Delete Movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshMovieBtn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
